<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationControlWidget</class>
 <widget class="QWidget" name="AnimationControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="animationLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1" alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="controlGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="2" column="1">
       <widget class="QSpinBox" name="currentCycleSpinBox">
        <property name="toolTip">
         <string>Currently displayed cycle</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="playbackLayout">
        <item>
         <widget class="QToolButton" name="playbackStartStopToolButton">
          <property name="toolTip">
           <string>Play or pause the animation</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/images/images.qrc">
            <normaloff>:/icon_pause.png</normaloff>
            <normalon>:/icon_play.png</normalon>:/icon_pause.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="currentCycleLabel">
        <property name="text">
         <string>Current Cycle</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Playback</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="timersGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="timersLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="delayIntervalLabel">
        <property name="text">
         <string>Delay Interval</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="cycleIntervalLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Cycle Interval</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="cycleIntervalSlider">
        <property name="toolTip">
         <string>Amount of time between animation steps</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>6000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="delayIntervalSlider">
        <property name="toolTip">
         <string>The delay before starting the next animation cycle</string>
        </property>
        <property name="maximum">
         <number>6000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="cycleTimeEdit">
        <property name="toolTip">
         <string>Uses the format &quot;minutes:seconds.milliseconds&quot;</string>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MinuteSection</enum>
        </property>
        <property name="displayFormat">
         <string>mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="delayTimeEdit">
        <property name="toolTip">
         <string>Uses the format &quot;minutes:seconds.milliseconds&quot;</string>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MinuteSection</enum>
        </property>
        <property name="displayFormat">
         <string>mm:ss.zzz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" alignment="Qt::AlignLeft|Qt::AlignTop">
    <widget class="QGroupBox" name="advancedSettingsGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="advancedSettingsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="baseSettingsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="baseSettingsLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="1" column="2">
       <widget class="QCheckBox" name="fadeCheckBox">
        <property name="toolTip">
         <string>Smoothes the animation</string>
        </property>
        <property name="text">
         <string>Fade</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="paletteComboBox">
        <property name="minimumSize">
         <size>
          <width>102</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Color scheme used in the animation</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="centerLayout">
        <item>
         <widget class="QSpinBox" name="centerXSpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="centerYSpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="reverseCheckBox">
        <property name="toolTip">
         <string>Play the animation in reverse</string>
        </property>
        <property name="text">
         <string>Reverse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Palette</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="paletteEditButton">
        <property name="text">
         <string>Edit Palettes...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="orientationComboBox">
        <property name="toolTip">
         <string>Direction that the animation displays in</string>
        </property>
        <item>
         <property name="text">
          <string>Horizontal</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images/images.qrc">
           <normaloff>:/icon_arrow_right.png</normaloff>:/icon_arrow_right.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images/images.qrc">
           <normaloff>:/icon_arrow_up.png</normaloff>:/icon_arrow_up.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horizontal (Flipped)</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images/images.qrc">
           <normaloff>:/icon_arrow_left.png</normaloff>:/icon_arrow_left.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical (Flipped)</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/images/images.qrc">
           <normaloff>:/icon_arrow_down.png</normaloff>:/icon_arrow_down.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="typeComboBox">
        <property name="minimumSize">
         <size>
          <width>102</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Type of animation to display</string>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blink</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-blink.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-blink.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cycle</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-cycle.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-cycle.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fire</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-fire.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-fire.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lightning</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-lightning.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-lightning.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mandelbrot</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-mandelbrot.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-mandelbrot.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plasma</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-plasma.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-plasma.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Radial</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-radial.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-radial.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-random.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-random.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-solid.png</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-solid.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sparkle</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-sparkle.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-sparkle.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wave</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-wave.gif</normaloff>../../lib/PixelMaestro/docsrc/images/animations/animation-wave.gif</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="centerLabel">
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="orientationLabel">
        <property name="text">
         <string>Orientation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeComboBox</tabstop>
  <tabstop>reverseCheckBox</tabstop>
  <tabstop>orientationComboBox</tabstop>
  <tabstop>fadeCheckBox</tabstop>
  <tabstop>paletteComboBox</tabstop>
  <tabstop>paletteEditButton</tabstop>
  <tabstop>centerXSpinBox</tabstop>
  <tabstop>centerYSpinBox</tabstop>
  <tabstop>playbackStartStopToolButton</tabstop>
  <tabstop>currentCycleSpinBox</tabstop>
  <tabstop>cycleTimeEdit</tabstop>
  <tabstop>cycleIntervalSlider</tabstop>
  <tabstop>delayTimeEdit</tabstop>
  <tabstop>delayIntervalSlider</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
